e2e-tests/data-sharded/data

```diff
@@ -0,0 +1,2002 @@
+[
```

Comment:
No need to commit actual data, just create a `data.js` file like this:
```
use app;
var bulk = db.city.initializeUnorderedBulkOp();
for(var i=0; i<200000; i++){
   bulk.insert( { "name": "city-"+i, "zipcode": i });
}
bulk.execute();
```
and then load it with the client.

Reply:
The problem is that my script inserts data in bulk (2000 records per request). And is't already not fast operation.
If start to insert 200000 records one by one this test will be slower almost 2000 times.

Reply:
https://github.com/percona/percona-server-mongodb-operator/commit/0ef77690fcb99f70a404dee504f3552e1c48a1ec